tests/shared.rs

```diff
@@ -193,6 +194,9 @@ pub fn assert_non_match_output(fixture_dir_name: &str, command_and_output: &str)
         .success()
         .stdout(predicate::function(|stdout: &str| {
             let stdout = massage_error_output(stdout);
+            if stdout != output {
+                print_diff(&stdout, &output, " ");
```

Comment:
This was a hacky approximation of a "pretty assertion diff" that I'm kind of inclined to keep in (it only kicks in if the test is failing)?

Reply:
What happens when color command codes are in the diff?